Modules/Core/QuadEdgeMesh/test/itkQuadEdgeMeshEulerOperatorCreateCenterVertexTest.cxx

```diff
@@ -74,7 +74,7 @@ itkQuadEdgeMeshEulerOperatorCreateCenterVertexTest(int, char *[])
   std::cout << "OK" << std::endl;
 #endif
 
-  (void)createCenterVertex->GetNameOfClass();
+  [[maybe_unused]] const std::string name_of_class = createCenterVertex->GetNameOfClass();
```

Comment:
- Maybe do an `ITK_TEST_EXPECT_EQUAL` here as well, following pull request #4583 ?

Reply:
Instead of introducing an unused variable, `name_of_class`, what about making it a "real" test? At least you could do something like:

    ITK_TEST_EXPECT_TRUE(createCenterVertex->GetNameOfClass());

Which tests that GetNameOfClass() returns a valid character string, not a null pointer.